scripts/devops_tasks/git_helper.py

```diff
@@ -28,7 +28,8 @@
     'azure-storage-file-datalake': '12.2.0b1',
     'azure-communication-identity': '1.0.0',
     'azure-communication-phonenumbers': '1.0.0',
-    'azure-communication-sms': '1.0.0'
+    'azure-communication-sms': '1.0.0',
+    'azure-search-documents': '11.2.0b3',
```

Comment:
To make sure we are on same page, we want to skip 11.0.0, 11.1.0 and all betas. We want to start with 11.2.0. Is this correct?

Reply:
@xiangyan99 that's exactly what this does. First allowed version is `11.2.0`.